portal/templates/coredata.html

```diff
@@ -79,12 +78,13 @@ <h3 class="tnth-headline">{{ _("More About You") }}</h3>
     <div class="error-continue error-message"></div>
     <br/><br/><br/><br/>
 {% endif %}
+<div id="coreDataError" class="error-message"></div>
 <input type="hidden" id="coreDataUserId" value="{{user.id if user}}" />
 <input type="hidden" id="procReturnAddress" value="{{ return_address | safe }}" />
 <input type="hidden" id="procAPIUrl" value="{{url_for('coredata_api.acquire')}}" />
 {% endblock %}
 {% block footer %}
-    <footer id="core_data_footer"></footer>
+    <footer id="core_data_footer">{{footer(user=user)}}</footer>
```

Comment:
Does the footer require a user object now? Would prefer to get that info from an API or sessionStorage, if possible